inc/admin/dashboard/namespace.php

```diff
@@ -240,39 +238,33 @@ function pending_invitations_callback() {
  * Callback for /wp-admin/user/ widget
  */
 function lowly_user_callback() {
-	echo '<p>' . __( 'Welcome to Pressbooks!', 'pressbooks' ) . '</p>';
+	echo '<p>' . sprintf( esc_html__( 'Welcome to %s', 'pressbooks' ), get_bloginfo( 'name', 'display' ) ) . '!</p>';
 	$user_has_books = count( get_blogs_of_user( get_current_user_id() ) ) > 1;
 	if ( ! $user_has_books ) {
-		echo '<p>' . __( 'You do not have access to any books at the moment.', 'pressbooks' ) . '</p>';
+		echo '<p>' . esc_html__( 'You do not currently have access to any books on this network.', 'pressbooks' ) . '</p>';
 	}
 	$contact = \Pressbooks\Utility\main_contact_email();
 	// Values can be 'all', 'none', 'blog', or 'user', @see wp-signup.php
 	$active_signup = apply_filters( 'wpmu_active_signup', get_site_option( 'registration', 'none' ) );
 	if ( in_array( $active_signup, [ 'none', 'user' ], true ) ) {
-		echo '<p>';
-		_e( 'This network does not allow users to create new books. To create a new book, please contact your Pressbooks Network Manager', 'pressbooks' );
-		if ( ! empty( $contact ) ) {
-			echo ' ' . __( 'at', 'pressbooks' ) . " $contact";
-		} else {
-			echo '.';
+		echo '<p>' . esc_html__( 'This network does not allow users to create new books. To create a new book, please contact a network manager', 'pressbooks' );
+		if ( ! empty( $contact ) && strpos( $contact, '@pressbooks.com' ) === false ) {
+			echo ' ' . esc_html__( 'at ', 'pressbooks' ) . sprintf( '<a href="mailto:%1$s">%2$s</a>', $contact, $contact );
```

Comment:
To avoid using the same variable twice you can do the following:

```php
echo ' ' . esc_html__( 'at ', 'pressbooks' ) . sprintf( '<a href="mailto:%1$s">%1$s</a>', $contact );
```

By using the same placeholder it will replace with the right variable.